discovery/db_discovery.go

```diff
@@ -60,7 +64,7 @@ func (dbo *DBOrchestratorPoolCache) GetURLs() []*url.URL {
 }
 
 func (dbo *DBOrchestratorPoolCache) GetOrchestrators(numOrchestrators int) ([]*net.OrchestratorInfo, error) {
-	orchs, err := dbo.node.Database.SelectOrchs()
+	orchs, err := dbo.node.Database.FilterOrchs(server.BroadcastCfg.MaxPrice())
```

Comment:
My interpretation was that it's used to return the number of O's in the dbCache regardelss of filters, that's how it's used in tests at least.

Reply:
I agree with @j0sh - I think it makes sense to interpret `Size()` as the size of the orchestrator pool that can be selected. In this change, while all Os are cached, we're only selecting from Os that pass the filter.

Reply:
The old `SelectOrch` *does* filter out rows based on date; it doesn't return all rows unconditionally.

If you look at the [actual usage](https://github.com/livepeer/go-livepeer/blob/master/server/broadcast.go#L162) of the Size function then it becomes clear we should be maintaining the same parameters, since the pool size is used to determine how many Os to select (and we don't want to over-select for Os that we shouldn't be using).